src/main/java/org/xwiki/contrib/authentication/jdbc/internal/BcryptPasswordHasher.java

```diff
@@ -0,0 +1,24 @@
+package org.xwiki.contrib.authentication.jdbc.internal;
```

Comment:
This file should have the same header than other Java files (should be just a copy/paste).